### ISSUE TITLE
Main README file has incorrect inforamtion

### ISSUE THREAD

**Issue by nherriot** [CJWorkbench/channels_rabbitmq#47]
Main 'readme' states:

```
Does not support [Worker and Background Tasks](https://channels.readthedocs.io/en/stable/topics/worker.html). (See https://github.com/CJWorkbench/channels_rabbitmq/pull/11#issuecomment-499185070 and use await get_channel_layer().current_connection to send to job queues.)
```

Specificaly when it says:

     *and use*
``` 
await get_channel_layer().current_connection 
```
     *to send to job queues.*

However, there is no attribute: **current_connection**
Perhaps a simple tutorial like this: ( https://testdriven.io/blog/django-channels/  ) would help developers?

```
    def __init__(
        self,
        host="amqp://guest:guest@127.0.0.1/asgi",
        local_capacity=100,
        remote_capacity=100,
        expiry=60,
        local_expiry: Optional[float] = None,
        group_expiry=None,
        ssl_context=None,
        groups_exchange="groups",
    ):
        self.host = host
        self.connect_timeout = 10.0  # seconds
        self.local_capacity = local_capacity
        self.remote_capacity = remote_capacity
        self.expiry = expiry
        self.local_expiry = local_expiry
        self.ssl_context = ssl_context
        self.groups_exchange = groups_exchange

        try:
            asyncio.get_running_loop()

```

**Comment by adamhooper** [comment]
Thanks! I think it should be `carehare_connection`, not `current_connection`.

PR welcome -- I'm on my phone and won't use my computer for days. And I welcome any extra documentation, certainly!

**Comment by adamhooper** [comment]
Closed by commit 1d18c4c
